core/src/main/java/hudson/slaves/OfflineCause.java

```diff
@@ -155,4 +155,14 @@ public ByCLI(String message) {
             this.message = message;
         }
     }
+
+    /**
+     * Caused by idle period.
```

Comment:
Description should be descriptive imho. `Offline caused by idle period` or `Offline caused by idle timeout`